app/controllers/state_file/questions/ny_w2_controller.rb

```diff
@@ -9,14 +9,21 @@ def self.show?(intake)
       end
 
       def index
-        @w2s_with_metadata = @w2s.map do |w2|
-          dfw2 = w2.state_file_intake.direct_file_data.w2s[w2.w2_index]
-          {
-            w2: w2,
-            employer_name: dfw2.EmployerName,
-            wages_amount: dfw2.WagesAmt,
-          }
+        if @w2s.length == 1
```

Comment:
So this renders the one W2 if there's only one and and index of the W2s if there are more?

Reply:
Correct!